apps/mwiniimporter/importer.cpp

```diff
@@ -834,35 +864,78 @@ void MwIniImporter::importGameFiles(multistrmap &cfg, const multistrmap &ini, co
     const boost::filesystem::path gameFilesDir(iniFilename.parent_path() /= "Data Files");
 
     multistrmap::const_iterator it = ini.begin();
-    for(int i=0; it != ini.end(); i++) {
+    for (int i=0; it != ini.end(); i++)
+    {
         gameFile = baseGameFile;
         gameFile.append(this->numberToString(i));
 
         it = ini.find(gameFile);
-        if(it == ini.end()) {
+        if(it == ini.end())
             break;
-        }
 
-        for(std::vector<std::string>::const_iterator entry = it->second.begin(); entry!=it->second.end(); ++entry) {
+        for(std::vector<std::string>::const_iterator entry = it->second.begin(); entry!=it->second.end(); ++entry)
+        {
             std::string filetype(entry->substr(entry->length()-3));
             Misc::StringUtils::lowerCaseInPlace(filetype);
 
-            if(filetype.compare("esm") == 0 || filetype.compare("esp") == 0) {
+            if(filetype.compare("esm") == 0 || filetype.compare("esp") == 0)
+            {
                 boost::filesystem::path filepath(gameFilesDir);
                 filepath /= *entry;
-                contentFiles.push_back(std::make_pair(lastWriteTime(filepath, defaultTime), *entry));
+                contentFiles.push_back({lastWriteTime(filepath, defaultTime), filepath});
             }
         }
     }
 
     cfg.erase("content");
     cfg.insert( std::make_pair("content", std::vector<std::string>() ) );
 
-    // this will sort files by time order first, then alphabetical (maybe), I suspect non ASCII filenames will be stuffed.
+    // sort by timestamp
     sort(contentFiles.begin(), contentFiles.end());
-    for(std::vector<std::pair<std::time_t, std::string> >::const_iterator iter=contentFiles.begin(); iter!=contentFiles.end(); ++iter) {
-        cfg["content"].push_back(iter->second);
+
+    MwIniImporter::deplist unsorted_files;
+
+    ESM::ESMReader reader;
```

Comment:
Running reader without setting an encoding. Not sure if that would work properly.